api/status.go

```diff
@@ -35,9 +36,12 @@ func (s *NodeService) GetStatus(r *http.Request, args *interface{}, requestBody
 	ctx, inslog := inslogger.WithTraceField(context.Background(), traceID)
 
 	inslog.Infof("[ NodeService.GetStatus ] Incoming request: %s", r.RequestURI)
-
 	statusReply := s.runner.NetworkStatus.GetNetworkStatus()
 
+	if !s.runner.cfg.IsAdmin {
```

Comment:
Move this check above, so we didn't call `s.runner.NetworkStatus.GetNetworkStatus()` if we not admin